internal/oapispec/routes.go

```diff
@@ -53,12 +53,14 @@ type Route struct {
 	JSONInputSchema func(ctx context.Context) string
 	// JSONOutputValue is a function that returns a pointer to a structure to take JSON output
 	JSONOutputValue func() interface{}
-	// JSONOutputCode is the success response code
-	JSONOutputCode int
+	// JSONOutputCodes is the success response code
+	JSONOutputCodes []int
```

Comment:
Should this simply be `OutputCodes` (there's nothing really JSON about it). And does it only list success codes, or does it support errors too?

Side note: seems like this openapi route stuff could almost be a separate package altogether.